starkbank/boleto/__boleto.py

```diff
@@ -97,18 +97,19 @@ def get(id, user=None):
     return rest.get_id(resource=_resource, id=id, user=user)
 
 
-def pdf(id, layout=None, user=None):
+def pdf(id, layout=None, hiddenFields=None, user=None):
     """# Retrieve a specific Boleto pdf file
     Receive a single Boleto pdf file generated in the Stark Bank API by passing its id.
     ## Parameters (required):
     - id [string]: object unique id. ex: "5656565656565656"
     ## Parameters (optional):
     - layout [string]: Layout specification. Available options are "default" and "booklet"
+    - hiddenFields [string]: List of string fields to be hidden in Boleto pdf. ex: ["customer Address"]
```

Comment:
`- hiddenFields [list of strings]`
`ex: ["customerAddress"]`

Reply:
ok